docs/src/contributing/testnet.md

```diff
@@ -0,0 +1,266 @@
+```bash
+$ gcloud auth login
+```
+If you are running the scripts on a headless machine, you can use curl to issue requests to confirm your auth.
+
+If you are doing it the first time, you might need to set up project
+```bash
+gcloud config set project principal-lane-200702
+```
+
+#### AWS
+Obtain your credentials from the AWS IAM Console and configure the AWS CLI with
+```bash
+$ aws configure
+```
+More information on AWS CLI configuration can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-quick-configuration)
+
+## Metrics configuration (Optional)
+Metrics collection relies on 2 environment variables that are patched to the remote nodes by net.sh:
+ * `RUST_LOG` to enable metrics reporting in principle
+ * `SOLANA_METRICS_CONFIG` to tell agave where to log the metrics
+
+### Preparation
+> [!NOTE]
+> Anza employees should follow the guide in notion to set up the influxDB account.
+
+ * Ensure that `${host}` is the host name of the InfluxDB you can access, for example `https://internal-metrics.solana.com:8086`
+ * Ensure that `${user}` is the name of an InfluxDB user account with enough
+rights to create a new InfluxDB database, for example `solana`.
+
+### To set up the metrics
+You will normally only need to do this once. Once this is done, you will be able to save the metrics configuration and load it later from the environment.
+
+* Go to ./net/ in agave repo
+* Run `./init-metrics.sh -c testnet-dev-${user} ${user} `
+  * Script will ask for a password, it is the same one you’ve created when making a user in the InfluxDB UI
+  * Put the username you have used in preparation, not your login user name
+  * If you need to set influxDb host, edit the script
+* The script will configure the database (recreating one if necessary) and append a config line in the very end of `net/config/config` file like the following:
+  * `export SOLANA_METRICS_CONFIG="host=${host},db=testnet-dev-${user},u=${user},p=some_secret"`
+  * You can store that line somewhere and append it to the config file when you need to reuse the database.
+  * You can also store it into your shell’s environment so you can run `./init-metrics.sh -e` to quickly load it
+  * Alternatively, you'll need to run `./init-metrics.sh` with appropriate arguments every time you set up a new cluster
+* Assuming no errors, your influxDB setup is now done.
+* For simple cases, storing `SOLANA_METRICS_CONFIG` in your env is appropriate, but you may want to use different databases for different runs of net.sh
+  * You can call ./init-metrics.sh before you call net.sh start, this will change the metrics config for a particular run.
+  * You can manually write `SOLANA_METRICS_CONFIG` in the `./net/config/config` file
+* By default, metrics are only logged by agave if `RUST_LOG` is set to `info` or higher. You can provide it as environment for `./net.sh start` command, or set this in your shell environment.
+  ```bash
```

Comment:
this particular RUST_LOG is sort of one I typically use when need to investigate transaction execution. I think for general public it is enough to have smth like `RUST_LOG=solana=info`

Reply:
solana_metrics=info is sufficient for metrics. edited as appropriate.